[Thu Jan 23 10:59:59.706312 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 10:59:59.706312 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215:   File "I:\\cgi\\access_manager.py", line 55, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 10:59:59.707306 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py
[Thu Jan 23 10:59:59.707306 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 10:59:59.707306 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py
[Thu Jan 23 10:59:59.914164 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 10:59:59.914164 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215:   File "I:\\cgi\\access_manager.py", line 55, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 10:59:59.914164 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 10:59:59.914164 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 10:59:59.914164 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55207] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:52.848063 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:52.848063 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:52.848063 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:52.848063 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:52.848063 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:53.989265 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:53.989265 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:53.990265 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:53.990265 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:53.990265 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.428999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.428999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.428999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.428999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.428999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.989890 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.989890 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.989890 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.989890 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:00:54.989890 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55208] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:01:04.611547 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:04.612548 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:04.612548 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:04.612548 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:04.612548 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:04.820266 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/123
[Thu Jan 23 11:01:04.820266 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/123
[Thu Jan 23 11:01:04.820266 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py, referer: http://localhost:8080/123
[Thu Jan 23 11:01:04.820266 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/123
[Thu Jan 23 11:01:04.820266 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py, referer: http://localhost:8080/123
[Thu Jan 23 11:01:06.918931 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:01:06.918931 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:01:06.918931 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:01:06.918931 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:01:06.918931 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55217] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:01:28.569026 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:28.569026 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:28.569026 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:28.569026 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:28.570023 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py
[Thu Jan 23 11:01:28.773414 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:01:28.773414 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215:   File "I:\\cgi\\access_manager.py", line 57, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:01:28.774436 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215:     controller_name = ucfirst(controller) + ucfirst(category) + "Controller"\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:01:28.774436 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215:                       ^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:01:28.774436 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55219] AH01215: NameError: name 'ucfirst' is not defined\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:08:17.575163 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55263] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:17.575163 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55263] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:17.575163 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55263] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:17.575163 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55263] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:17.575163 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55263] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:17.575163 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55263] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:17.575163 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55263] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:24.404474 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:24.404474 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:24.404474 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:24.404474 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:24.404474 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:24.404474 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:24.404474 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:27.728871 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:27.728871 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:27.928580 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:08:27.928580 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:08:27.928580 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:08:27.928580 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:08:27.928580 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:08:27.928580 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:08:27.928580 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55264] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:08:52.917944 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55266] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:52.917944 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55266] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:52.917944 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55266] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:52.917944 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55266] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:52.917944 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55266] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:52.917944 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55266] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:08:52.917944 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55266] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:45.457559 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:45.457559 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:45.673139 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:09:45.673139 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:09:45.673139 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:09:45.673139 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:09:45.673139 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:09:45.673139 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:09:45.673139 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:09:49.401566 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:49.401566 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:49.401566 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:49.401566 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:49.401566 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:49.401566 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:49.401566 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py
[Thu Jan 23 11:09:49.608140 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:09:49.608140 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:09:49.608140 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:09:49.608140 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:09:49.608140 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:09:49.608140 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:09:49.608140 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55269] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:25.774833 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:25.774833 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:25.774833 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:25.774833 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:25.774833 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:25.774833 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:25.774833 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:26.120775 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:26.121775 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:26.121775 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:26.121775 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:26.121775 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:26.121775 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:26.121775 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:29.274702 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:29.275697 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:29.489430 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:10:29.489430 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:10:29.489430 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:10:29.489430 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:10:29.489430 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:10:29.489430 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:10:29.489430 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py, referer: http://localhost:8080/
[Thu Jan 23 11:10:32.276193 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:32.276193 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:32.276193 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:32.276193 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:32.276193 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:32.276193 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:32.276193 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py
[Thu Jan 23 11:10:32.496808 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:32.496808 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:32.496808 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:32.496808 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:32.496808 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:32.497798 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:10:32.497798 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:55274] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:11:04.955453 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py
[Thu Jan 23 11:11:04.955453 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:11:04.955453 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py
[Thu Jan 23 11:11:04.955453 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py
[Thu Jan 23 11:11:04.955453 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py
[Thu Jan 23 11:11:04.955453 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py
[Thu Jan 23 11:11:04.955453 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py
[Thu Jan 23 11:11:05.166536 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215: I:\\cgi\\access_manager.py:56: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:11:05.167539 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215:   print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:11:05.167539 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215: Traceback (most recent call last):\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:11:05.167539 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215:   File "I:\\cgi\\access_manager.py", line 56, in <module>\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:11:05.167539 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215:     print("Contriller: %s, Action: %s, Slug: %s" (controller,action, slug))\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:11:05.167539 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
[Thu Jan 23 11:11:05.167539 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:55283] AH01215: TypeError: 'str' object is not callable\r: I:/cgi/access_manager.py, referer: http://localhost:8080/index.html
